pyproject.toml

```diff
@@ -36,7 +36,7 @@ dependencies = [
     "spherical-geometry>=1.2.22",
     "stcal @ git+https://github.com/spacetelescope/stcal.git@main",
     "stdatamodels @ git+https://github.com/spacetelescope/stdatamodels.git@main",
-    "stpipe>=0.7.0,<0.8.0",
+    "stpipe @ git+https://github.com/braingram/stpipe.git@model_to_exptype",
```

Comment:
Will be updated to `@main` after https://github.com/spacetelescope/stpipe/pull/201 is merged.

Reply:
Updated to main in c751b4ed3d58babe1ed066fbd59c4957a745803b